ElasticBox/boxes-api.md

```diff
@@ -1,73 +1,1770 @@
+    "profile": {
+      "location": "ap-northeast-1",
+      "schema": "http://elasticbox.net/schemas/aws/cloudformation/profile"
+    },
+    "schema": "http://elasticbox.net/schemas/boxes/cloudformation",
+    "updated": "2015-10-28 13:46:55.247211",
+    "automatic_updates": "off",
+    "requirements": [
+
+    ],
+    "description": "CF test",
+    "created": "2015-10-27 12:11:24.734064",
+    "deleted": null,
+    "variables": [
+      {
+        "required": false,
+        "type": "Binding",
+        "name": "Binding",
+        "value": "5030fc49-773a-42fa-b569-c551e4f90d77",
+        "visibility": "private"
+      }
+    ],
+    "provider_id": "7e841966-1dec-4460-a981-1db4e1eec10c",
+    "visibility": "workspace",
+    "members": [
+
+    ],
+    "template": {
+      "url": "/services/blobs/download/5630d1cf14841250525226a6/template.json",
+      "upload_date": "2015-10-28 13:46:55.186875",
+      "length": 104,
+      "content_type": "text/x-shellscript"
+    },
+    "owner": "operations",
+    "organization": "elasticbox",
+    "type": "CloudFormation Service",
+    "id": "91645eed-173f-4aac-a713-69c6be7582fe",
+    "name": "CF WordPress 2"
+  },
+  "updated": "2015-10-28 13:47:15.567441",
+  "automatic_updates": "off",
+  "name": "CF WordPress 2",
+  "service": {
+    "type": "CloudFormation Service",
+    "id": "eb-5cn45",
+    "machines": [
+
+    ]
+  },
+  "tags": [
+
+  ],
+  "deleted": null,
+  "variables": [
+
+  ],
+  "created": "2015-10-28 13:47:15.567441",
+  "state": "processing",
+  "uri": "/services/instances/i-ywf1hu",
+  "boxes": [
+    {
+      "schema": "http://elasticbox.net/schemas/boxes/cloudformation",
+      "updated": "2015-10-28 13:46:55.247211",
+      "automatic_updates": "off",
+      "requirements": [
+
+      ],
+      "description": "CF test",
+      "created": "2015-10-27 12:11:24.734064",
+      "deleted": null,
+      "variables": [
+        {
+          "required": false,
+          "type": "Binding",
+          "name": "Binding",
+          "value": "5030fc49-773a-42fa-b569-c551e4f90d77",
+          "visibility": "private"
+        }
+      ],
+      "visibility": "workspace",
+      "members": [
+
+      ],
+      "template": {
+        "url": "/services/blobs/download/5630d1cf14841250525226a6/template.json",
+        "upload_date": "2015-10-28 13:46:55.186875",
+        "length": 104,
+        "content_type": "text/x-shellscript"
+      },
+      "owner": "operations",
+      "organization": "elasticbox",
+      "type": "CloudFormation Service",
+      "id": "91645eed-173f-4aac-a713-69c6be7582fe",
+      "name": "CF WordPress 2"
+    }
+  ],
+  "members": [
+
+  ],
+  "bindings": [
+
+  ],
+  "owner": "operations",
+  "operation": {
+    "event": "deploy",
+    "workspace": "operations",
+    "created": "2015-10-28 13:47:15.564698"
+  },
+  "schema": "http://elasticbox.net/schemas/instance",
+  "id": "i-ywf1hu",
+  "icon": null
+}
+```
+### Example: Update a CloudFormation Stack in Real-Time
+ +
```

Comment:
@brandysmith this particular line and the following does have a `+` in the text.  It should show up in Atom.  Please double-check